src/MenuItem/MenuItem.js

```diff
@@ -59,6 +59,10 @@ class MenuItem extends Component {
 
   static propTypes = {
     /**
+     * Override the default anchorOrigin children used.
```

Comment:
We need to improve the wording. We should refer to the nested menu at least.
Here is what we have on the Popover:
```js
    /**
     * This is the point on the anchor where the popover's
     * `targetOrigin` will attach to.
     * Options:
     * vertical: [top, center, bottom]
     * horizontal: [left, middle, right].
     */
```